File: src/api/dotnet/ReExpr.cs
/*++
Copyright (<c>) 2016 Microsoft Corporation

Module Name:

    ReExpr.cs

Abstract:

    Z3 Managed API: Regular Expressions

Author:

    Christoph Wintersteiger (cwinter) 2012-11-23

Notes:
    
--*/
using System.Diagnostics;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


namespace Microsoft.Z3
{
    /// <summary>
    /// Regular expression expressions
    /// </summary>
    public class ReExpr : Expr
    {
        #region Internal
        /// <summary> Constructor for ReExpr </summary>
        internal ReExpr(Context ctx, IntPtr obj)
            : base(ctx, obj)
        {
            Debug.Assert(ctx != null);
        }
        #endregion
    }
}

File: src/api/java/ReExpr.java
/**
Copyright (c) 2012-2016 Microsoft Corporation
   
Module Name:

    ReExpr.java

Abstract:

Author:

    @author Christoph Wintersteiger (cwinter) 2012-03-15

Notes:
    
**/ 

package com.microsoft.z3;

/**
 * Re expressions
 **/
public class ReExpr<R extends Sort> extends Expr<ReSort<R>>
{
    /**
     * Constructor for ReExpr
     * @throws Z3Exception on error
     **/
    ReExpr(Context ctx, long obj)
    {
        super(ctx, obj);
    }
}
